ee/modules/650-runtime-audit-engine/docs/README.md

```diff
@@ -0,0 +1,107 @@
+
+![falco daemonset](../../images/650-runtime-audit-engine/falco_daemonset.png)
+
+> NOTE: To achieve the maximum level of security, Falco is recommended to be deployed as a systemd unit.
+> However, a Kubernetes cluster with the autoscaling feature enabled makes it harder to operate. 
+> Additional security measurements such as multitenancy and admission policy control mechanisms provide the required level of security to mitigate attacks on the Falco DaemonSet.
+
+A single Pod consists of five containers:
+![falco pod](../../images/650-runtime-audit-engine/falco_pod.png)
+
+1. `falco-driver-loader`: init container that compiles eBPF program and stores it into empty dir to provide access to it for Falco.
+2. `falco`: collects events, enriches them with metadata and stores them.
+3. `rules-loader`: collects custom resources (FalcoAuditRules) from Kubernetes and store them in a shared directory (empty dir).
+4. `falcosidekick`: only exports events as metrics to be able to alert on them.
+5. `kube-rbac-proxy`: protects falcosidekick metrics endpoint.
+
+## Audit Rules
+
+An ability to collect events on its own means nothing, because the amount of data that can be exported from a Linux kernel is too much to analyze by a human brain.
+Rules aimed to solve this problem and collect only events according to certain pattens implied to detect any suspicious activity.
+
+The main part of a rule is a condition expression (for which the [conditions syntax](https://falco.org/docs/rules/conditions/) is used).
+
+### Embedded rules
+
+There is a pair of rule files included that cannot be switched off. 
+These rules are aimed to highlight Deckhouse security problems and problems connected to the runtime-audit-engine module itself.
+
+- `/etc/falco/falco_rules.yaml` - syscall rules
+- `/etc/falco/k8s_audit_rules.yaml` - Kubernetes audit rules
+
+
+### Custom audit rules
+
+To add custom security audit rules, users can use a custom resource definition `FalcoAuditRules.` 
+Each Falco agent pod has a sidecar container with a [shell-operator](https://github.com/flant/shell-operator) instance.
+This sidecar reads rules from custom resources and stores it on Pod's filesystem in `/etc/falco/rules.d/` folder.
+Falco will automatically reload the configuration once a new rule appears.
+
+![falco shell-operator](../../images/650-runtime-audit-engine/falco_shop.png)
+
+This schema allows to use the IaC approach to maintain Falco rules.
+
+## Requirements
+
+### OS
+
+The module uses the eBPF Falco driver to ingest syscall data. It works better for environments where loading a kernel module is untrusted or not supported, e.g., GKE, EKS, and other Managed Kubernetes solutions.
+Yet there are known limitations for the eBPF driver:
+* The eBPF probe may not work for every system.
+* At least Linux kernel version 4.14 is required, but the Falco project suggests an LTS kernel of 4.14/4.19 or above.
```

Comment:
Should we add init container which checks kernel version  like in node-local-dns module ?

Reply:
Honestly, I didn't check how falco works in every environment. This requirement is copied from the Falco doc.

Reply:
If falco uses bpf minimal kernel version is 4.16. This is not a blocking point, may be add this to TODO ?

Reply:
Added to TODO list